modules/rules_fuzzing/0.5.1/presubmit.yml

```diff
@@ -0,0 +1,24 @@
+matrix:
+  platform: ["macos", "ubuntu2204"]
+  bazel: ["6.x", "7.x"]
+
+tasks:
+  build_targets:
+    name: "Build targets"
+    bazel: ${{ bazel }}
+    platform: ${{ platform }}
+    build_targets:
+      - "@rules_fuzzing//fuzzing/..."
```

Comment:
@mmorel-35 You can specify target patterns with a leading `-` here to exclude them.